Chess/king.h

```diff
@@ -0,0 +1,16 @@
+#ifndef H_KING
+#define H_KING
+#include "piece.h"
+class King:public Piece{
+
+public:
+    King(int row, int column);
+    virtual void makeMove(int difRow, int difColumn);
+protected:
+    virtual bool isValidMove(int difRow, int difColumn);
+private:
+    int row;
+    int column;
```

Comment:
row si column le mostenim din Piece, nu o sa fie nevoie sa fie si aici. Acum am vazut ca mi-a scapat asta in alte clase de piese :)) O sa le repar eu dincolo